src/main/api/2/studio-api-2.yaml

```diff
@@ -3056,25 +3288,830 @@ paths:
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
+
+  /dashboard/workflow:
+    get:
+      tags:
+        - dashboard
+      summary: Get content for dashboard of workflow
+      description: 'Required permission "get_dashboard" ("site member")'
+      operationId: getDashboardWorkflow
+      parameters:
+        - name: siteId
+          in: query
+          description: site ID
+          required: true
+          schema:
+            type: string
+        - name: sortBy
+          in: query
+          description: >-
+            order result by column
+            List of columns:
+              - label
+              - path
+              - modifier
+              - scheduledDate
+              - environment
+              - content type
+          required: true
+          schema:
+            type: string
+        - name: order
+          in: query
+          description: order of sorted data (ASC or DESC)
+          schema:
+            type: string
+            format: ASC, DESC
+        - name: groupBy
+          in: query
+          description: group data by column
+          schema:
+            type: string
+        - name: offset
+          in: query
+          description: offset of the first result item, default 0
+          schema:
+            type: integer
+        - name: limit
+          in: query
+          description: number of results to return, default 10
+          schema:
+            type: integer
+        - name: filters
+          in: query
+          description: >-
+            list of filters to apply on result set
+            Filter:
+              - path
+              - modifier
+              - scheduledDateFrom
+              - scheduledDateTo
+              - content type
+              - environment
+              - state
+          schema:
+            type: array
+            items:
+              type: object
+              properties:
+                column:
+                  type: string
+                value:
+                  type: string
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  response:
+                    $ref: '#/components/schemas/ApiResponse'
+                  item:
+                    $ref: '#/components/schemas/DashboardItem'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
+
+  /dashboard/user:
```

Comment:
Should this go under `/user/search` <== Just asking so we're clear on our approach. This is not a site API, it's a global admin/user management API.